packages/opentelemetry-exporter-collector/src/transform.ts

```diff
@@ -0,0 +1,241 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { hexToBase64, hrTimeToTimeStamp } from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/tracing';
+import { Attributes, Link, TimedEvent, TraceState } from '@opentelemetry/types';
+import * as collectorTypes from './types';
+
+const OT_MAX_STRING_LENGTH = 128;
+const OT_MAX_ATTRIBUTES = 30;
+
+const LINK_TYPE_UNSPECIFIED: collectorTypes.LinkTypeUnspecified = 0;
+// const LINK_TYPE_CHILD_LINKED_SPAN: collectorTypes.LinkTypeChildLinkedSpan = 1;
```

Comment:
Why is this commented out?

Reply:
because the specs defines 3 types, but now this one is not being used yet. Correct me if I'm wrong but we don't have such information yet. I have added "detection" on the params I have and I can be sure at 100% it is here
https://github.com/open-telemetry/opentelemetry-js/blob/c09bc5caf62f13ce9eb6471749512cdd25a464f1/packages/opentelemetry-exporter-collector/src/transform.ts#L159

Reply:
Could you create an issue for make the third one supported to match the spec?

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/568